content/tutorials/manage-the-rippled-server/configuration/configure-advisory-deletion.md

```diff
@@ -0,0 +1,98 @@
+# Configure Advisory Deletion
+
+The default config file sets `rippled` to automatically delete outdated history of XRP Ledger state and transactions as new ledger versions become available. If your server uses most of its hardware resources during peak hours, you can configure the server to delete ledgers only when prompted by a command scheduled to run during off-peak hours to reduce the chances of online deletion.
```

Comment:
"to run during off-peak hours to reduce the chances of online deletion"... doing what? I assume it's something like "running during peak hours", but the sentence reads awkwardly to me.

Reply:
Maybe “to run during off-peak hours to reduce the performance impact of online deletion.”

Reply:
Good catch. This was a copy-edit failure. Changed to:

> If your server uses most of its hardware resources during peak hours, you can configure the server to delete ledgers only when prompted by a command scheduled to run during off-peak hours, so that online deletion is less likely to impact server performance.